packages/cli/src/zosfiles/delete/zfs/zfs.definition.ts

```diff
@@ -47,12 +47,23 @@ export const ZfsDefinition: ICommandDefinition = {
             description: strings.OPTIONS.FOR_SURE,
             type: "boolean",
             required: true
+        },
+        {
+            name: "ignore-not-found",
+            aliases: ["i", "ignore-nf"],
```

Comment:
This comment applies to other option definitions.

To be consistent with our [rules for naming commands](https://github.com/zowe/zowe-cli/blob/master/docs/CommandFormatStandards.md#naming-commands) (and options), I think that instead of `ignore-nf`, it should be `inf`
We can keep the `i` if we want people to use `-if` when deleting datasets and files 😋 

```suggestion
            aliases: ["i", "inf"],
```